core/msg.c

```diff
@@ -93,10 +95,12 @@ static int _msg_send(msg_t *m, kernel_pid_t target_pid, bool block, unsigned sta
         return -1;
     }
 
+    thread_t *me = (thread_t *) sched_active_thread;
```

Comment:
Can you briefly explain the incentive of using `me` instead of `sched_active_thread` in the following code? Is `sched_active_thread` changing and you just need to cache the former value?